proto/cosmos/app/module/v1/module.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package cosmos.app.module.v1;
```

Comment:
I'm a bit confused why this is called `app`? Shouldn't it be like `cosmos.example_module.module.v1`?

Reply:
This is the app module itself. Basically the module which manages app configs

Reply:
Ah right got it. Why is it in v1, but its Query and Msg services in v1alpha1?